core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -223,8 +224,12 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
         for (category in RankingType.values()) {
             val headers = Table().apply { defaults().pad(5f) }
-            headers.add(category.name.replace('_', ' ').toLabel()).row()
-            headers.addSeparator().fillX()
+            val textAndIcon = Table().apply { defaults().pad(5f) }
```

Comment:
No need to defaults if you explicitly set 6 of the 8 contained paddings? Also, doesn't that make overall padding 10f larger than before?

Reply:
That's a bit overzealous with padding, plus it led to a visual peculiarity for the demographics screen where the leftmost column's separator wasn't at the same height as the other separators. Removed and simplified.